bindings/rascal/representations/spherical_covariants.py

```diff
@@ -81,17 +82,13 @@ class documentation
             inversion_symmetry=inversion_symmetry,
             lam=lam)
 
-        cutoff_function = dict(
-            type=cutoff_function_type,
-            cutoff=dict(
-                value=interaction_cutoff,
-                unit='AA'
-            ),
-            smooth_width=dict(
-                value=cutoff_smooth_width,
-                unit='AA'
-            ),
+        cutoff_function_parameters.update(
+            **dict(interaction_cutoff=interaction_cutoff,
```

Comment:
I think the `**dict` syntax is unnecessary here -- it first gathers the keywords into a dict, then explodes them back out into keywords. If I'm not mistaken, you should be able to just remove `**dict([...])` and keep everything inside, to the same effect.

Reply:
Also, is it possible to just pass these parameters along to SphericalExpansion and let the parsing be done once there, rather than three times (once in each of the Spherical* representations)?

Reply:
SphericalExpansion is directly used at the c++ level so you need to make the hypers dict in every calculators.